SwagMediaSftp.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/*
+ * (c) shopware AG <info@shopware.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace SwagMediaSftp;
+
+require __DIR__ . "/vendor/autoload.php";
```

Comment:
why is this necessary?

Reply:
needed for this lib "league/flysystem-sftp"

Reply:
is not used here. should be in the subscriber file, shouldn't it?

Reply:
I think it make no real difference but It could be a better place there :-)